NitroxPatcher/Patches/Dynamic/PDAScanner_Scan_Patch.cs

```diff
@@ -53,6 +63,77 @@ public static void Callback()
             }
         }
 
+        public static readonly int PACKET_SENDING_RATE = 500; // in milliseconds
+
+        // Tuple<LatestProgressTime, LatestPacketSendTime, Entry>
+        public static Dictionary<TechType, Tuple<DateTimeOffset, DateTimeOffset, PDAScanner.Entry>> ThrottlingEntries = new Dictionary<TechType, Tuple<DateTimeOffset, DateTimeOffset, PDAScanner.Entry>>();
+        // Store the thread concerning a certain type
+        public static Dictionary<TechType, Thread> ThrottlingThreads = new Dictionary<TechType, Thread>();
```

Comment:
Won't it be possible to combine the tuple and the thread in one class so you can have only one dictionary?

Reply:
Sounds good to me, I'll make this change